consutil/main.py

```diff
@@ -11,7 +11,9 @@
     import pexpect
     import re
     import subprocess
+    import sys
     from tabulate import tabulate
+    from lib import *
```

Comment:
Why this import?

Reply:
File "/usr/lib/python2.7/dist-packages/consutil/main.py", line 24, in consutil
sys.exit(1)
NameError: global name 'sys' is not defined

Reply:
The 'main.py' need to call function which is defined in 'lib.py'.

Reply:
Please add more detail. Which function is defined in lib.py?

Reply:
> Please add more detail. Which function is defined in lib.py?

Such as 'run_command', 'getAllDevices','getBusyDevices','getConnectionInfo','getLineNumber','checkDevice', all of them defined in 'lib.py'.

Reply:
Thanks for the info. I forgot there was a file named "lib.py" in the consutil source. That filename is very nondescript; I think we should rename it in the future. Maybe something like "consutil_common.py" or similar.